scripts/RUN_2AM.sh

```
@@ -59,8 +59,18 @@ python fetch_power.py --year=$(date --date '7 days ago' +'%Y') --domain=conus
 python fetch_power.py --year=$(date --date '7 days ago' +'%Y') --domain=china
 python fetch_power.py --year=$(date --date '7 days ago' +'%Y') --domain=europe
 python fetch_power.py --year=$(date --date '7 days ago' +'%Y') --domain=sa
+# Force a ~5 month old data
+python fetch_power.py --date=$(date --date '150 days ago' +'%Y-%m-%d') --domain=conus
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)